students/carlos_novoa/lesson09/mailroom.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python3
+import os
+import datetime
+
+now = datetime.datetime.now()
+
+"""
+Lesson9 - Object Oriented Mailroom
+"""
+
+_donors = {
+    'Jim Halpert': [1.00],
+    'Pam Beesley': [1000.00, 2000.00, 3000.00],
+    'Dwight Shrute': [2.00, 3.00],
+    'Michael Scott': [10.00, 20.00, 30.00],
+    'Andy Bernard': [500.00]
+}
+
+
+class Helpers():
+    def format_currency_str(amount=None):
+        return "${0:.2f}".format(float(amount))
+
+
+class Donors:
+    donors = None
+
+    def __init__(self, donors=None):
+        self.donors = donors if donors else _donors
```

Comment:
It would be better to make donors be always be an instance attribute created from the `__init__` parameter, rather than allowing the attribute to default to the value of a global variable. One of the main reasons for using classes is to encapsulate related data (attributes) and actions/behavior (methods) that use that data inside the class